content/terraform/v1.12.x/docs/language/values/outputs.mdx

```diff
@@ -0,0 +1,117 @@
+---
+page_title: Use outputs to expose module data
+description: Learn how to output data from a module to expose information about your infrastructure on the CLI, in HCP Terraform, and to other Terraform configurations.
+---
+
+# Use outputs to expose module data
+
+> **Hands-on:** Try the [Output data from Terraform](/terraform/tutorials/configuration-language/outputs) tutorial.
+
+Outputs let you expose information about your infrastructure on the command line, in HCP Terraform, and in other Terraform configurations. The `output` block serves the following purposes in Terraform:
+
+- Child modules can expose resource attributes to parent modules.
+- Root modules can display values in CLI output.
+- Other Terraform configurations using [remote state](/terraform/language/state/remote) can access root module outputs with the `terraform_remote_state` data source.
+- Pass information from a Terraform operation to an automation tool.
+
+## Define outputs
+
+Add `output` blocks to export information about your module's infrastructure. For example, you can add two `output` blocks to expose the ID and the IP address of your configuration's web server:
+
+<CodeBlockConfig highlight="1,6" hideClipboard>
+
+```hcl
+output "instance_id" {
+  description = "ID of the EC2 instance"
+  value       = aws_instance.web.id
+}
+
+output "instance_ip" {
+  description = "Private IP address of the EC2 instance"
+  value       = aws_instance.web.private_ip
+}
+
+resource "aws_instance" "web" {
+  # …
+}
+```
+
+</CodeBlockConfig>
+
+You can set the `value` argument of an `output` block to any valid expression. To learn more, refer to the [`output` block reference](/terraform/language/block/output).
+
+
+## Access output values
+
+Depending where you define an output value, you can access the information a module exposes in different ways. Terraform displays root module output values in the CLI after you apply your configuration. If you are using HCP Terraform, your workspace's overview page also lists your configuration's outputs.
+
+### Child module outputs
+
+Defining an `output` block in a child module exposes that value to the parent module. You can use outputs to pass a value from a child module to a parent module. Parent modules can access child module outputs using `module.<CHILD_MODULE_NAME>.<OUTPUT_NAME>` syntax.
```

Comment:
```suggestion
Defining an `output` block in a child module exposes that value to the parent module so that you can pass values from a child module to a parent module. Parent modules can access child module outputs using `module.<CHILD_MODULE_NAME>.<OUTPUT_NAME>` syntax.
```
These first two sentences are saying almost the same thing.